### PR TITLE
Introduce a 23video slide-type

### PR DESCRIPTION
**Pull request by danquah**
OSD-62

### CODE REVIEW

File: src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="twentythree" style="position: relative">
```
Comment (tripox) [inline src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.html:1]: Seems duplicate - already in the styling.

File: src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    <!-- https://www.twentythree.net/help/embedding-your-videos -->
+    <div style="width:100%; height:100%; position: relative;">
```
Comment (tripox) [inline src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.html]: Consider putting the inline styling in the stylesheet if possible

File: src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            region.$timeout(function () {
+                // Build the url for the video.
+                slide.url = "//video.kk.dk/v.ihtml/player.html?source=embed&photo%5fid=" + slide.options.id + "&autoPlay=1&loop=1";
+                slide.url += "&showDescriptions=0";
+                slide.url += "&showLogo=0";
+                slide.url += "&socialSharing=0";
+                slide.url += "&hideBigPlay=1";
+                slide.url += "&showTray=0";
+                slide.url += "&defaultQuality=fullhd";
+                if (slide.options.sound === 1) {
```
Comment (tripox) [inline src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.js]: Should this not be 0? 
The checkbox is enabling sound, not disabling.

File: src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+         * @param slide
+         *   The slide.
+         * @param region
+         *   The region object.
+         */
+        run: function runSlide (slide, region) {
+            // Wait fadeTime before start to account for fade in.
+            region.$timeout(function () {
+                // Build the url for the video.
+                slide.url = "//video.kk.dk/v.ihtml/player.html?source=embed&photo%5fid=" + slide.options.id + "&autoPlay=1&loop=1";
```
Comment (tripox) [inline src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.js]: Seems very specific for kk.dk?
Shouldn't it be configurable?

**Review by tripox — CHANGES_REQUESTED** [review]

**Comment by danquah** [comment]
@tripox did a fair bit of rework and have (amongst other things) addressed most of your comments. Could I have a re-review pretty please? :)

**Review by tripox — APPROVED** [review]

**Pull request merged**